supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/GoogleLabels.scala

```diff
@@ -18,16 +18,19 @@ final case class GoogleLabel(key: String, value: String)
 object GoogleLabels {
 
   val MaxLabelLength = 63
-  val GoogleLabelRegexPattern = "[a-z]([-a-z0-9]*[a-z0-9])?"
-  val GoogleLabelRegex = GoogleLabelRegexPattern.r
+  val GoogleLabelKeyRegexPattern = "[a-z]([_-a-z0-9]*[a-z0-9])?"
```

Comment:
```suggestion
  val GoogleLabelKeyRegexPattern = "[a-z]([-_a-z0-9]*[a-z0-9])?"
```

Reply:
Dashes would either need to be escaped or moved to a terminal position within the character class to avoid invoking their "meta" meaning. https://stackoverflow.com/a/3697210

Reply:
I did not realize that `-` took effect between any pair of characters. Thanks!